### PR TITLE
data:EminTalhaArik

### PR DESCRIPTION
**Pull request by EminTalhaArik**

### CODE REVIEW

File: data/EminTalhaArik.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{
+  "name": "Emin Arık",
+  "type": "personal",
+  "bio": "Business Analyst | Software Developer | **Full Time Learner**",
+
+  "tags": [
+    "Entrepreneurship",
+    "Analysis",
+    "Python",
+    "Django",
```
Comment (khushalarora) [inline data/EminTalhaArik.json:10]: Please remove the extra **' Comma '** from this line :+1: 
```Suggestion
    "Django"
```

File: data/EminTalhaArik.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      "group": "Social Media",
+      "name": "LinkedIn",
+      "url": "https://linkedin.com/in/emin-talha-arik/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "group": "Social Media",
+      "name": "Instagram",
+      "url": "https://instagram.com/talhemin",
+      "icon": "FaInstagram",
```
Comment (khushalarora) [inline data/EminTalhaArik.json:28]: Please remove the extra **' Comma '** from this line :+1: 
```Suggestion
      "icon": "FaInstagram"
```

File: data/EminTalhaArik.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    },
+    {
+      "group": "Social Media",
+      "name": "Twitter",
+      "url": "https://twitter.com/talheminn",
+      "icon": "FaTwitter",
```
Comment (khushalarora) [inline data/EminTalhaArik.json:34]: Please remove the extra **' Comma '** from this line :+1: 
```Suggestion
      "icon": "FaTwitter"
```

File: data/EminTalhaArik.json [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+      "name": "YouTube Shorts: Quick technical tips",
+      "url": "https://youtube.com/eddiejaoudetv",
+      "icon": "FaYoutube"
+    },
+    {
+      "group": "Business",
+      "name": "TEINN | Teens In Innovation",
+      "url": "http://teinn.org",
+      "icon": "FaLink"
+    },
```
Comment (khushalarora) [inline data/EminTalhaArik.json:59]: Please remove the extra **' Comma '** from this line :+1: 
```Suggestion
    }
```

File: data/EminTalhaArik.json [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  ],
```
Comment (khushalarora) [inline data/EminTalhaArik.json:60]: Please remove the extra **' Comma '** from this line :+1: 
```Suggestion
  ]
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @EminTalhaArik, thank you for contributing :tada:

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**